pkg/cli/completion_test.go

```diff
@@ -129,13 +129,29 @@ func TestAppsThenContainersCompletion(t *testing.T) {
 }
 
 func TestAppsCompletion(t *testing.T) {
+	var DefaultImageList = []apiv1.Image{{
+		TypeMeta:   metav1.TypeMeta{},
```

Comment:
nit: You don't need this like if `TypeMeta` is getting set to its "zero" value. Here and other tests altered in these changes.

Reply:
yeah, removed now